src/k8s/pkg/utils/cidr.go

```diff
@@ -37,3 +72,28 @@ func GetKubernetesServiceIPsFromServiceCIDRs(serviceCIDR string) ([]net.IP, erro
 	}
 	return firstIPs, nil
 }
+
+// ParseAddressString parses an address string and returns a canonical network address.
+func ParseAddressString(address string, port int64) (string, error) {
```

Comment:
Please add some basic unittests for this function

Reply:
I've added some unit tests for this function. However, since we need to access the network interfaces from the host running the tests, I've implemented a workaround for this specific scenario. Briefly, the workaround is to pull the defaultAddress using the LXD util and infer a /24 CIDR for that IP Address. Please let me know what you think. Thanks @bschimke95!

Reply:
sounds good to me - thanks!